src/spaces/grid.jl

```diff
@@ -41,11 +42,14 @@ cartesian indices have Euclidean distance `≤ r` from the cartesian index of th
 position.
 
 An example using `GridSpace` is the [Forest fire model](@ref).
+
+TODO pathfinder
```

Comment:
Here the simplest is to reference the `[Path-finding](@ref)` section of the docs and the `AStar` struct.